fizzBuzzRedux.js

```diff
@@ -0,0 +1,60 @@
+//Benton Green
+(function() {
+  function FizzBuzz(){
+    var startNum;
+    var endNum;
+  }
+    FizzBuzz.prototype.readFizzBuzz = function() {
+      console.log("read");
+      startNum = parseInt(document.getElementById('startNumber').value);
+      endNum = parseInt(document.getElementById('endNumber').value);
+    }
+
+    FizzBuzz.prototype.writeFizzBuzz = function(startingNum, endingNum) {
```

Comment:
The `start` and `end` parameters were supposed to be on the `read()` method, not the `write()`. As written, your `read()` method doesn't do anything useful; lines 15 and 16 should really be in the `readFizzBuzz` method.